### DISCUSSION TITLE
series data - non time stamped

### DISCUSSION THREAD

**Discussion by andrewcztrack** [ourownstory/neural_prophet#218] [Q&A - get help using NeuralProphet] [answered]
hi @ourownstory !!!

love the library! it super cool!

Just wondering if there are plans to implement non time series data. e.g if you have a single nx1 numpy array, create forecasts 30 steps ahead.

Kind regards and thanks,
Andrew

**Comment by ourownstory** [comment]
Hi Andrew @andrewcztrack,
Thank you!
Do you mind explaining your specific problem setting a little closer?

From what I understand, you are talking about tabular data, where the predicted vector is a forecast. 
For this application, sklearn and fastai have excellent support.

Or do you mean a time series without specific time stamps?

**Reply by andrewcztrack** [reply]
Hi @ourownstory !! Thank you for the reply. I can clarify that I mean time series with out a specific time stamp.
Kind Regards and thanks,
Andrew

**Comment by ourownstory** [comment, marked as answer]
> time series with out a specific time stamp.

Yes, @andrewcztrack that has been on the timeline for a while now (tracked in #18). For now, you can create a dummy-series of time-stamps and add it to your data as `"ds"`.
